lib/model.g.dart

```diff
@@ -24,19 +24,23 @@ class Config {
       ..tabs = List_fromData<Tab>(data['tabs'], Tab.fromData);
   }
 
-  static void listen(DocStorage docStorage, ConfigCollectionListener listener, String collectionRoot) =>
-      listenForUpdates<Config>(docStorage, listener, collectionRoot, Config.fromSnapshot);
+  static void listen(DocStorage docStorage, ConfigCollectionListener listener,
+          String collectionRoot) =>
```

Comment:
The changes in this file seem to be done automatically by your editor - I think it's set to enforce a 80 characters per line limit? Feel free to keep it in if you prefer, by I find with Dart that that tends to be a bit too small, so my personal preference is to increase it to 100 or 120.

Reply:
Yep 80 is too restrictive, will apply this in another PR. Updating it to 120 now causes too many changes in other files.